allensdk/brain_observatory/behavior/project_apis/data_io/behavior_project_cloud_api.py

```diff
@@ -0,0 +1,146 @@
+        cache.load_latest_manifest()
+        return BehaviorProjectCloudApi(cache)
+
+    def get_behavior_only_session_data(
+            self, behavior_session_id: int) -> BehaviorSession:
+        """get a BehaviorSession by specifying behavior_session_id
+
+        Parameters
+        ----------
+        behavior_session_id: int
+            the id of the behavior_session
+
+        Returns
+        -------
+        BehaviorSession
+
+        """
+        row = self._behavior_only_session_table.query(
+                f"behavior_session_id=={behavior_session_id}")
+        if row.shape[0] != 1:
+            raise RuntimeError("The behavior_only_session_table should have "
+                               "1 and only 1 entry for a given "
+                               "behavior_session_id. For "
+                               f"{behavior_session_id} "
+                               f" there are {row.shape[0]} entries.")
+        data_path = self.cache.download_data(row.data_file_id.values[0])
+        return BehaviorSession.from_nwb_path(str(data_path))
+
+    def get_session_data(self, ophys_session_id: int) -> BehaviorOphysSession:
+        """get a BehaviorOphysSession by specifying session_id
+
+        Parameters
+        ----------
+        ophys_session_id: int
+            the id of the ophys_session
+
+        Returns
+        -------
+        BehaviorOphysSession
+
+        """
+        row = self._session_table.query(
+                f"ophys_session_id=={ophys_session_id}")
+        if row.shape[0] != 1:
+            raise RuntimeError("The behavior_session_table should have "
+                               "1 and only 1 entry for a given "
+                               f"ophys_session_id. For {ophys_session_id} "
+                               f" there are {row.shape[0]} entries.")
+        data_path = self.cache.download_data(row.data_file_id.values[0])
+        return BehaviorOphysSession.from_nwb_path(str(data_path))
+
+    def _get_session_table(self):
+        session_table_path = self.cache.download_metadata(
+                "ophys_session_table.csv")
+        self._session_table = pd.read_csv(session_table_path)
+
+    def get_session_table(self) -> pd.DataFrame:
+        """Return a pd.Dataframe table with all ophys_session_ids and relevant
+        metadata."""
+        return self._session_table
+
+    def _get_behavior_only_session_table(self):
+        session_table_path = self.cache.download_metadata(
+                "behavior_session_table.csv")
+        self._behavior_only_session_table = pd.read_csv(session_table_path)
+
+    def get_behavior_only_session_table(self) -> pd.DataFrame:
+        """Return a pd.Dataframe table with all ophys_session_ids and relevant
+        metadata."""
+        return self._behavior_only_session_table
+
+    def _get_experiment_table(self):
+        experiment_table_path = self.cache.download_metadata(
+                "behavior_session_table.csv")
```

Comment:
I think you meant "ophys_experiment_table.csv"

Reply:
fixed